app/views/contact_forms/response.html.erb

```diff
@@ -1,4 +0,0 @@
-<h1>Contact Us</h1>
```

Comment:
@mcritchlow - Based on the Machform I created for DMR before, the response message was created with the Machform respond message option.